Tests/ForageSDKTests/ForageSentryTest.swift

```diff
@@ -0,0 +1,51 @@
+
+import XCTest
+@testable import ForageSDK
+
+struct Event {
+    let message: String
+}
+
+protocol SentryClient {
+    func capture(event: Event)
+}
+class MockSentryClient: SentryClient {
+    var capturedEvents: [Event] = []
+    
+    func capture(event: Event) {
+        capturedEvents.append(event)
+    }
+}
+
+class EventLogger {
+    private let sentryClient: SentryClient
+    
+    init(sentryClient: SentryClient) {
+        self.sentryClient = sentryClient
+    }
+    
+    func logEvent(message: String) {
+        let event = Event(message: message)
+        sentryClient.capture(event: event)
+    }
+}
+
+final class ForageSentryTest: XCTestCase {
+    var forageMocks: ForageMocks!
+    
+    override func setUp() {
+        ForageSDK.setup(ForageSDK.Config(environment: .sandbox))
+        ForageSDK.shared.service = nil
+        forageMocks = ForageMocks()
+    }
```

Comment:
Hey hey! I was wondering if you could provide some clarity about how the `setUp` function is involved with the `testEventLoggerSendsToSentry` test case? My understanding of `testEventLoggerSendsToSentry` is that it may not directly depend on `ForageSDK` or `forageMocks`?

Reply:
@devinmorgan I will be removing this test since it isnt testing the SentrySDK.start and is instead testing a mocked sentry client